tests/pages/adminPage.js

```diff
@@ -175,4 +177,35 @@ module.exports = {
     I.pressKey(['Control', 'a']);
     I.pressKey('Backspace');
   },
+
+  /**
+   * Encapsulates Tooltip data verification.
+   * There could be only one tooltip popup on a page.
+   *
+   * @param   tooltipObj        one of {@link pmmSettingsPage.tooltips} or an object with similar structure
+   * tooltipObj have to have field tooltipText containing selector for tooltip text and field tooltipReadMoreLink
+   * containing selector for tooltip doc link
+   * @returns {Promise<void>}   requires await in test body.
+   */
+  async verifyTooltip(tooltipObj) {
+    const tooltipIcon = tooltipObj.iconLocator;
+    const tooltipText = tooltipObj.tooltipText.as('Tooltip text');
+
+    I.waitForVisible(tooltipIcon, 5);
+    I.moveCursorTo(tooltipIcon);
+    I.scrollTo(tooltipText);
+    I.waitForVisible(tooltipText, 5);
+    I.seeTextEquals(tooltipObj.text, tooltipText);
+    /* there are tooltip without "Read more" link */
+    if (tooltipObj.link) {
```

Comment:
this will cause "cannot ready property of unrefined" error. Please add `Object.hasOwn(tooltipObj, 'link')` first.

Reply:
![image](https://user-images.githubusercontent.com/104006070/209393445-a27066e3-7c61-47b8-a58c-6e1fbc8c575b.png)
I don't think that's the case